### ISSUE TITLE
Should we add Cloudflare?

### ISSUE THREAD

**Issue by veb** [codecraft-dunedin/codecraft-dunedin.github.io#1]
I think it'd be a good idea to add cloudflare to our website https://codecraft.org.nz as it shows some decent analytics. 

I use it on my personal blog, which is also Jekyll and hosted via gh-pages. 

Some examples from mine:

![image](https://user-images.githubusercontent.com/376245/96039815-c4523480-0ec5-11eb-896d-a67c80a8e84f.png)

And:

![image](https://user-images.githubusercontent.com/376245/96039851-d46a1400-0ec5-11eb-8d27-85806b1b028d.png)

It's very simple analytics but does the job for me. The only thing we'd need to do is point the domain to Cloudflare, then point the dns to gh-pages where the site is hosted. Adding a site on Cloudflare is very simple, and usually will copy the DNS records that already exist as well.

If anyone else has any better suggestions, let us know!

**Comment by howbazaar** [comment]
Sounds like a good idea to me. I should let more people know about the repo to get more commentary.

**Comment by thenickdude** [comment]
It's basically irrelevant for our site, but in general CloudFlare is a huge privacy hazard and the web would be better without it. 

We're unlikely to make decisions based on the analytics anyway I reckon?

**Comment by thomir** [comment]
Cloudflare are also well-known supporters of some pretty unpleasant hate groups online. I'd rather not support them in any way.

**Comment by haszari** [comment]
Cloudflare is more than just analytics, in particular it's a proxy for providing CDN and https. We get super fast hosting (possibly a GitHub CDN?) + https for free from GitHub pages, so I don't think there's a need for Cloudflare.

Analytics could well be useful - would be interesting to know how the site is being used, etc. There are lots of options for free good analytics, [Google Analytics](https://marketingplatform.google.com/about/analytics/) is probably the most popular option.

**Comment by veb** [comment]
I was hoping to stay away from Google Analytics, but everything you said is definitely true @haszari  - and Cloudflare does have servers hosted here in NZ makes things much faster, but you're right we do get that from gh-pages to a lesser degree.

If people are really against Cloudflare... even when I feel the "they support hate groups" is blown well out of proportion, then I feel it's easier to move on without it. Opinions differ, but in all honesty it's not the biggest issue in the world.

I'm against using Google Analytics, not the biggest fan of being tracked everywhere. It would be nice to know who is using the site, why, and where they're coming from. Gives us some good insights. Perhaps there's another option?

**Comment by ianrrees** [comment]
I agree with @thenickdude's point about making decisions based on analytics.  To paraphrase Hamming, "you get what you measure", and ultimately I think we're aiming for in-person engagement.

What sort of content do we want on the site?  I'd be happy with a calendar for upcoming events, and perhaps relevant links to related groups, slides from previous talks, etc.
